README.md

```diff
@@ -117,6 +120,32 @@ val transformed = stream.through(transform[IO, Json](selector, json => Json.obj(
 println(transformed.through(values[IO, Json]).compile.toList.unsafeRunSync())
 ```
 
+### Patches
+
+The `fs2-data-json-diffson` module provides some integration with [diffson][diffson].
+It allows for patching a Json stream as it is read to emit the patched value downstream.
```

Comment:
As it a rather advanced thing, maybe add a use case example (doesn't have to be code, just one or two sentences why this is a cool thing to do)?

Reply:
I added a note. Thanks for the suggestions. If you have any other idea, then I'd be glad to hear them.